galaxy_ng/app/settings.py

```diff
@@ -112,3 +114,79 @@
 # Disable django guardian anonymous user
 # https://django-guardian.readthedocs.io/en/stable/configuration.html#anonymous-user-name
 ANONYMOUS_USER_NAME = None
+
+GALAXY_ENABLE_API_ACCESS_LOG = os.getenv('GALAXY_ENABLE_API_ACCESS_LOG', default=False)
```

Comment:
You probably want to default it to True, temporarily until the installer is fixed.

```suggestion
GALAXY_ENABLE_API_ACCESS_LOG = os.getenv('GALAXY_ENABLE_API_ACCESS_LOG', default=True)
```